pkg/apis/v1alpha1/types.go

```diff
@@ -76,6 +80,23 @@ type MonitoringStackStatus struct {
 	// ??
 }
 
+type PrometheusConfig struct {
+	// Define persistent volume claim for prometheus
+	// +optional
+	PersistentVolumeClaim *PersistentVolumeClaim `json:"persistentVolumeClaim,omitempty"`
```

Comment:
I would use a `corev1.PersistentVolumeClaimSpec` directly here. As it is now, it's not possible for users to select volume claims by labels, which is a common use case for statefulsets

Reply:
Done